datashader/bokeh_ext.py

```diff
@@ -219,6 +234,10 @@ class InteractiveImage(object):
 
     def __init__(self, bokeh_plot, callback, delay=200, timeout=2000, throttle=None,
                  **kwargs):
+        warnings.warn('InteractiveImage has been deprecated as of 0.11.0. '
```

Comment:
It should say that it's been deprecated since 0.8.0, though at that time it was only in the release notes.